editor.planx.uk/src/ui/shared/InputRowLabel.tsx

```diff
@@ -7,9 +7,6 @@ const Label = styled(Typography)(({ theme }) => ({
   flexGrow: 0,
   paddingRight: theme.spacing(2),
   alignSelf: "center",
-  "&:not(:nth-child(1))": {
-    paddingLeft: theme.spacing(2),
-  },
```

Comment:
I can't see where the style applies, tried all usual tricks of solid red borders, hotpink background etc to locate when this should apply without any luck.